app/allocation/controllers/cancel.test.js

```diff
@@ -41,9 +41,9 @@ describe('Cancel controller', function() {
       })
 
       it('should cancel the allocation', function() {
-        expect(allocationService.cancel).to.be.calledWithExactly(
-          mockAllocation.id
-        )
+        expect(allocationService.cancel).to.be.calledWithExactly({
+          id: mockAllocation.id,
```

Comment:
Can we add a test to ensure this method is called with the correct data, and to ensure it removes the omitted values from the session model.